hack/generator/pkg/astmodel/allof_type.go

```diff
@@ -159,3 +159,19 @@ func (allOf *AllOfType) String() string {
 
 	return fmt.Sprintf("(allOf: %s)", strings.Join(subStrings, ", "))
 }
+
+// DebugDescription adds a description of the current AnyOf type to the passed builder
+// builder receives the full description, including nested types
+// types is a dictionary for resolving named types
+func (allOf *AllOfType) WriteDebugDescription(builder *strings.Builder, types Types) {
+	builder.WriteString("AllOf[")
+	first := true
```

Comment:
minor: You have an index in the `ForEach` func so you could just do away with this and check if `ix > 0` I think?

Reply:
This applies on other function implementations too but won't repeat comment there

Reply:
Heh. I'm too used to C#'s **foreach**. Will fix.

Reply:
Done.